app/kuma-dp/cmd/run.go

```diff
@@ -54,5 +54,12 @@ func newRunCmd() *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.PersistentFlags().StringVarP(&cfg.Dataplane.Name, "name", "", "", "Name of the Dataplane")
+	cmd.PersistentFlags().Uint32VarP(&cfg.Dataplane.AdminPort, "admin-port", "", 9901, "Port for Envoy Admin")
```

Comment:
Instead of defining default value again (`9901`), let's reuse value from `kuma_dp.DefaultConfig()`

E.g., 

```
cmd.PersistentFlags().Uint32Var(&cfg.Dataplane.AdminPort, "admin-port", cfg.Dataplane.AdminPort, "Port for Envoy Admin")
```